### ISSUE TITLE
Optimization for high-resolution generation

### ISSUE THREAD

**Issue by alfonslm** [lllyasviel/stable-diffusion-webui-forge#2236]
TLDR: high res generation can be optimised about 20% to 40%

I made some napkin math and wrote this post about it [High-res fix optimization](https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/2234) and would guess that the basic implementation without solving for the optimal n in the optimization algorithm should be easy enough to implement since the foundation for this is the high-res fix with more steps. 

Solving for the optimal n shouldn't be impossible either since it's already known how to plot the best estimate and some one have already done a time estimator from what I remember that should be good enough to get the constants needed to calculate the best n value.
